src/simulate.py

```diff
@@ -284,17 +286,23 @@ def simulate(self, computation_dfg, hw):
             # numeric layer value as a node attribute
             for node in nodes:
                 fake_hw.nodes[node]["layer"] = layer
-
         counter = 0
 
         generations = list(reversed(
             list(nx.topological_generations(nx.reverse(computation_dfg)))
         ))
+
+        place_n_route.export_graph(computation_dfg, "", "computation_dfg") # remove
+        place_n_route.export_graph(fake_hw, "", "fake_hw") # remove
+        place_n_route.export_graph(hw.parasitic_graph, "", "parasitic_graph") # remove
```

Comment:
what does export graph do and why are you exporting both a `dfg` and a hardware graph.